Add BZX384 zener diode

evanshultz opened the pull request:
https://assets.nexperia.com/documents/data-sheet/BZX384_SERIES.pdf

![image](https://user-images.githubusercontent.com/1936989/65554163-99252d80-dedd-11e9-9c94-e2acd791a7df.png)

The trailing `xx` is a little weird since there can be two or three characters for the voltage, but I kept it since that's how zener families are done in the library. And the zener tolerance is to be considered wildcarded as well. Let me know if you want any changes here.

---

All contributions to the kicad library must follow the [KiCad library convention](http://kicad-pcb.org/libraries/klc/)

Thanks for creating a pull request to contribute to the KiCad libraries! To speed up integration of your PR, please check the following items:

- [ ] Provide a URL to a datasheet for the symbol(s) you are contributing
- [ ] An example screenshot image is very helpful
- [ ] Ensure that the associated footprints match the [official footprint library](https://github.com/kicad/kicad-footprints)
  - A new fitting footprint must be submitted if the library does not yet contain one.
- [ ] If there are matching footprint PRs, provide link(s) as appropriate
- [ ] Check the output of the Travis automated check scripts - fix any errors as required
- [ ] Give a reason behind any intentional library convention rule violation.

cpresser commented:
- All other Zeners in the lib seem to have this exact drawing, so this one is good. While looking at this I also checked `Device:D_Zener` which has a slightly different drawing (one less poly-line, instead: longer pins meet in the center)
- Why did you not include the dash in the name? How about `BZX384-xxxx`. Four wildcards, since that covers the longest possible name. I would go for at least 3 wildcards, there is no device that has less than 3 characters after the dash.

evanshultz commented:
- I have cleaned up and commonalized FET symbols, but not other types yet. Thanks for the note. There are two other PRs by my which touch the diode library (one assigned to you) which I hoped to get merged before I cleaned up that library.
- Updated symbol name.
